manifests/kots-config.yaml

```diff
@@ -430,57 +439,104 @@ spec:
           type: text
           required: true
           when: '{{repl (ConfigOptionEquals "storageBucketsBehaviour" "custom_aws_s3") }}'
+        - name: storageBucketsAwsS3Region
+          title: Buckets AWS S3 Region
+          when: '{{repl (ConfigOptionEquals "storageBucketsBehaviour" "custom_aws_s3") }}'
+          help_text: Region of the buckets created in AWS S3
+          type: text
+          required: true
+
+        # Azure
         - name: storageBucketsAzureAccessKey
           title: AccessKey for Azure-Blob
           type: text
           required: true
           when: '{{repl (ConfigOptionEquals "storageBucketsBehaviour" "custom_azb") }}'
+
         - name: storageBucketsWorkspaceImports
-          title: Workspace Imports Bucket
-          help_text: Bucket name to be used for imports
+          title: Temp Bucket
+          help_text: Bucket name to be used for storing temporary files during imports
           type: text
           required: true
-          default: repl{{ if ConfigOptionEquals "replicatedLicenseType" "dev"}}repl{{ print (ConfigOption "derivedSelfHostedGCPProjectId") "-client-storage"}}repl{{ end }}
-        - name: storageBucketsWorkspaceImportsIsPublic
-          help_text: |-
-            Please select the access mode of this bucket
-          type: select_one
-          default: private
+          default: repl{{ if and (ConfigOptionEquals "replicatedLicenseType" "dev") (ConfigOptionEquals "storageBucketsBehaviour" "custom_gcs")}}repl{{ print (ConfigOption "derivedSelfHostedGCPProjectId") "-client-storage"}}repl{{ end }}
```

Comment:
I think that you need to add 2 conditions because in `kots-helm.yaml` you are not using the `optionalValues` block, right? :thinking:

Reply:
I added two because we want to put the default value just if we're in a dev license and using GCS. That's to use the default buckets that we provision with the SH 😄